plugins/libplugin.c

```diff
@@ -698,7 +700,9 @@ void plugin_log(enum log_level l, const char *fmt, ...)
 }
 
 void plugin_main(char *argv[],
-		 void (*init)(struct plugin_conn *rpc),
+		 void (*init)(struct plugin_conn *rpc,
```

Comment:
This change makes the commit break, hurting bisectability.  Presumably *this* particular change should be in the next commit.

Or I could just squash it into a single large commit...